platform_versioned_docs/version-23.4.0/data/data-explorer.mdx

```diff
@@ -88,6 +88,22 @@ Data Explorer supports file uploads to your private cloud storage buckets. From
 You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.
 :::
 
+### Download multiple files
+
+You can download multiple files via the user interface or using a script in your cloud provider's CLI. 
+
+:::note
+The browser download functionality currently does not work in non-Chromium browsers.
+:::
+
+Open the cloud bucket that you'd like to download and select the download icon in the **Actions** column. You can choose **Download selected** or **Download selected using code**. 
+
+The code snippet provided is specific to the cloud provider you've configured. You may be prompted to authenticate during the download process. Refer to your cloud provider's documentation for troubleshooting credential-related issues:
+
+- [GCP](https://cloud.google.com/sdk/gcloud/reference/storage)
+- [AWS](https://docs.aws.amazon.com/cli/latest/reference/s3/)
+- [Azure](https://learn.microsoft.com/en-us/cli/azure/storage?view=azure-cli-latest)
+
 ### AWS S3 CORS configuration
```

Comment:
There needs to be a CORS policy applied for multiple file downloads too. See the validation [tests](https://github.com/seqeralabs/tower-backlog/issues/1270).

Reply:
@robnewman, is this a request to incorporate those steps here? Or just to note that a user needs to deal with CORS policies somehow? Thanks!

Reply:
It would be helpful to have examples for sure.